src/components/PeopleFilters.tsx

```diff
@@ -2,17 +1,55 @@
+import { useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { getSearchWith } from '../utils/searchHelper';
+import { SearchLink } from './SearchLink';
+
+type Props = {};
+
+export const PeopleFilters: React.FC<Props> = ({}) => {
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const query = searchParams.get('query') || '';
+  const sexStatus = searchParams.get('sex');
+  const centuries = searchParams.getAll('centuries');
+
+  const handleQueryChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setSearchParams(
+      getSearchWith(searchParams, { query: event.target.value || null }),
+    );
```

Comment:
It's a good practice to destructure the properties you need from the event object right in the argument list. It would be `const handleQueryChange = ({ target: { value } }: React.ChangeEvent<HTMLInputElement>) => {` and inside the function body you can use `value` directly.